src/main/java/io/cdap/plugin/zendesk/source/batch/ZendeskBatchMultiSource.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright © 2020 Cask Data, Inc.
+ * Copyright © 2022 Cask Data, Inc.
```

Comment:
No need to update copyright for all files that have been modified. Just add copyright for any new files added in the PR

Reply:
done